macros/light/src/lib.rs

```diff
@@ -433,3 +434,11 @@ pub fn light_discriminator(input: TokenStream) -> TokenStream {
         .unwrap_or_else(|err| err.to_compile_error())
         .into()
 }
+
```

Comment:
Can we add an inline doc similar to the documentation for the `light_traits` and `light_accounts` macros?

The goal is: developers read only this and understand how to use it successfully in their applications.

Reply:
Yes, will do

Reply:
Done